### PR TITLE
UN-411: Insights to Stories rename app

### PR DESCRIPTION
**Pull request by TNA-Allan**
Related ticket(s):
https://national-archives.atlassian.net/browse/UN-411

## About these changes

Calls `rename_app` to update insights to stories in deployment and local environments
This update only works on first call, subsequent calls have not effect since the change has taken place.
However another PR post deployment will remove these changes.

## How to check these changes

This PR is required to be merged into `feature/UN-411-insight-to-story`.
Observe the logs for echoed output.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly myself before handing over to reviewer.
- [x] Included the ticket number in the PR title to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## For Reviewer

Once PR is merged, check and arrange to deploy on platform-sh.

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -287,6 +287,8 @@ def pull_database_from_platform(c, environment_name):
     )
 
     try:
+        print("Renaming app...")
+        run_management_command(c, "rename_app insights stories", check_returncode=True)
```
Comment (ababic) [inline fabfile.py:291]:
```suggestion
        # TODO: Rename command to be removed once applied in all local environments
        print("Renaming app...")
        run_management_command(c, "rename_app insights stories", check_returncode=True)
```

File: .platform.app.yaml [hunk 2]
```diff
@@ -59,6 +59,7 @@ hooks:
     python manage.py collectstatic --no-input
 
   deploy: |
+    python manage.py rename_app insights stories
```
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply .platform.app.yaml:62]: I like this, although we have a PR that removes this bit. Its a good practice to have that comment.
Thanks @ababic

File: fabfile.py (hunk 1 above)
Comment (TNA-Allan) [reply fabfile.py:291]: I like this, although we have a PR that removes this bit. Its a good practice to have that comment.
Good eye :) :) 
Thanks @ababic

File: .platform.app.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply .platform.app.yaml:63]: Done. Moved it above `deploy |` to enable comment.

File: fabfile.py (hunk 1 above)
Comment (TNA-Allan) [reply fabfile.py:292]: Done.

**Comment by TNA-Allan** [comment]
Closed - follow on[ PR-601](https://github.com/nationalarchives/ds-wagtail/pull/601)

**Pull request closed without merging**